lib/simp/beaker_helpers.rb

```diff
@@ -1170,62 +1192,57 @@ def install_puppet
-  # @raise if disable contains an invalid repo name.
+  # @param sut [Beaker::Host]  Host on which to configure SIMP repos
+  # @param disable [Array[String]] List of repos to disable
+  # @raise [StandardError] if disable contains an invalid repo name.
   #
   # Examples:
   #  install_simp_repos( myhost )           # install all the repos an enable them.
   #  install_simp_repos( myhost, ['simp'])  # install the repos but disable the simp repo.
   #
-  # Current set of valid SIMP repo names:
-  #  'simp'
-  #  'simp_deps'
+  # Valid repo names include any repository available on the system.
   #
-  def install_simp_repos(sut, disable = [] )
-    repos = {
-      'simp' => {
-        :baseurl   => 'https://packagecloud.io/simp-project/6_X/el/$releasever/$basearch',
-        :gpgkey    => ['https://raw.githubusercontent.com/NationalSecurityAgency/SIMP/master/GPGKEYS/RPM-GPG-KEY-SIMP',
-                    'https://download.simp-project.com/simp/GPGKEYS/RPM-GPG-KEY-SIMP-6'
-                     ],
-        :gpgcheck  => 1,
-        :sslverify => 1,
-        :sslcacert => '/etc/pki/tls/certs/ca-bundle.crt',
-        :metadata_expire => 300
-      },
-      'simp_deps' => {
-        :baseurl   => 'https://packagecloud.io/simp-project/6_X_Dependencies/el/$releasever/$basearch',
-        :gpgkey    => ['https://raw.githubusercontent.com/NationalSecurityAgency/SIMP/master/GPGKEYS/RPM-GPG-KEY-SIMP',
-                    'https://download.simp-project.com/simp/GPGKEYS/RPM-GPG-KEY-SIMP-6',
-                    'https://yum.puppet.com/RPM-GPG-KEY-puppetlabs',
-                    'https://yum.puppet.com/RPM-GPG-KEY-puppet',
-                    'https://apt.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG-96',
-                    'https://artifacts.elastic.co/GPG-KEY-elasticsearch',
-                    'https://grafanarel.s3.amazonaws.com/RPM-GPG-KEY-grafana',
-                    'https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-$releasever'
-                   ],
-        :gpgcheck  => 1,
-        :sslverify => 1,
-        :sslcacert => '/etc/pki/tls/certs/ca-bundle.crt',
-        :metadata_expire => 300
-      }
-    }
-    # Verify that the repos passed to disable are in the list of valid repos
-    disable.each { |d|
-      unless repos.has_key?(d)
-        raise("ERROR: install_simp_repo - disable contains invalid SIMP repo '#{d}'.")
+  # For backwards compatibility purposes, the following translations are
+  # automatically performed:
+  #
+  #  * 'simp'
+  #    * 'simp-community-simp'
+  #
+  #  * 'simp_deps'
+  #    * 'simp-community-epel'
+  #    * 'simp-community-postgres'
+  #    * 'simp-community-puppet'
+  #
+  def install_simp_repos(sut, disable = [])
+    on(sut, 'puppet resource package simp-release-community source="https://download.simp-project.com/simp-release-community.rpm" ensure=latest provider=rpm')
+
+    unless disable.empty?
+      if disable.include?('simp')
+        disable.delete('simp')
```

Comment:
should this code be modifying a passed parameter?

Reply:
@lnemsick-simp In this case it's correcting it and that change should probably be pushed back up. Would you like me to change it to a `dup`?